src/options.cpp

```diff
@@ -68,21 +68,19 @@ options_manager::options_manager() :
     general_page_( "general", to_translation( "General" ) ),
     interface_page_( "interface", to_translation( "Interface" ) ),
     graphics_page_( "graphics", to_translation( "Graphics" ) ),
-    debug_page_( "debug", to_translation( "Debug" ) ),
     world_default_page_( "world_default", to_translation( "World Defaults" ) ),
+    debug_page_( "debug", to_translation( "Debug" ) ),
     android_page_( "android", to_translation( "Android" ) )
 {
     pages_.emplace_back( general_page_ );
     pages_.emplace_back( interface_page_ );
     pages_.emplace_back( graphics_page_ );
     // when sharing maps only admin is allowed to change these.
-    if( !MAP_SHARING::isCompetitive() || MAP_SHARING::isAdmin() ) {
-        pages_.emplace_back( debug_page_ );
-    }
-    // when sharing maps only admin is allowed to change these.
     if( !MAP_SHARING::isCompetitive() || MAP_SHARING::isAdmin() ) {
         pages_.emplace_back( world_default_page_ );
+        pages_.emplace_back( debug_page_ );
```

Comment:
I don't think any change apart from this one is necessary.

Reply:
That is definitely right. The other changes are basically just moving the code around so that they _still_ are displayed in the same order as they pop up in the options tab. That might be unneccessary and I can remove it if you want - I just thought it would be a small QoL thing for us developers.

Reply:
I wasn't even aware that that was happening.  If there's no mechanism to enforce it, it's not worth doing.

I'd much prefer less code churn to maintaining a particular ordering in the code.